src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Link de login inválido ou expirado.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Muitas tentativas de login inválidas, por favor, tente novamente em %minutes% minuto.</target>
```

Comment:
We could write this one without "%minutes%", Maybe something like this: 

```
Demasiadas tentativas de login, tente novamente num minuto.
```

(I copied this from the Portuguese translations. I dont know if it make sense. But I hope you understand my point)

Reply:
In Brazilian Portuguese, "num" is usually for informal conversations, but I am going to update my PR with "em um minuto" rather than "em 1 minuto".